### PR TITLE
Restructure session sidebar and fix spacing

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

File: app/components/SessionSidebar.jsx [hunk 1]
```diff
@@ -6,16 +6,16 @@ import styles from './sessionSidebar.scss';
 
 class SessionSidebar extends Component {
 
+    getHandle(type) {
```
Comment (jebbinSCL) [inline app/components/SessionSidebar.jsx]: I think we will have an issue with this function if someone has multiple handles of a given type? e.g multiple emails or multiple phone numbers?

Comment (lclarksl) [inline app/components/SessionSidebar.jsx]: Yeah that's true. The UI doesn't deal with this at moment so I've selected the first one in the meantime.

Comment (jebbinSCL) [inline app/components/SessionSidebar.jsx]: That's fine then.

**Comment by jebbinSCL** [comment]
:shipit:

**Comment by ianlovell** [comment]
:shipit:

File: app/components/SessionSidebar.jsx (hunk 1 above)
Comment (lclarksl) [inline app/components/SessionSidebar.jsx]: I've added a story for it

Comment (csscottlogic) [inline app/components/SessionSidebar.jsx]: Good spot

**Pull request merged**